conformance/spec048_duckdb/testdata/no_manifest_action/README.md

```
@@ -0,0 +1 @@
+Deliberately missing dagu-action.yaml, to prove the action executor's error when a source action bundle has no manifest.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)